From 6b9f6e2e8beac05dac7d7023eb1559cc1e5fffe3 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 7 Aug 2014 13:43:38 +0000
Subject: [PATCH 0562/1076] MANTIS 0029957 ConveyorComp: CUSS1.3 Conveyor
 (without ConveyorAEA) CUSS1.3 errata interface ConveyorSBD {   /** If true,
 conveyor can only move in forward direction */   readonly attribute boolean
 onewayForward;

git-svn-id: svn://localhost/SelfServiceCommon/trunk@706 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Cuss/characteristics.idl | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/SelfServiceCommon/Massai/idl/Cuss/characteristics.idl b/SelfServiceCommon/Massai/idl/Cuss/characteristics.idl
index 3a661f21..2590860e 100644
--- a/SelfServiceCommon/Massai/idl/Cuss/characteristics.idl
+++ b/SelfServiceCommon/Massai/idl/Cuss/characteristics.idl
@@ -544,6 +544,9 @@ module Characteristics
         /** The maximum number of bags a conveyor can handle */
         readonly attribute long maxBags;
 
+        /** If true, conveyor can only move in forward direction */
+        readonly attribute boolean onewayForward;
+
       /** If true, conveyor has a security barrier (for user safety) */
       readonly attribute boolean barrierCapable;
 
-- 
2.41.0.windows.1

